.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+<!--
+     Before submitting a Pull Request, please ensure you've done the following:
+     - 📖 Read our Contributing Guide & Coding Standards: https://github.com/4chain-ag/code-standards-bsv/blob/main/.github/CODE_STANDARDS.md
+     - 📖 Read the short Code of Conduct: https://github.com/4chain-ag/code-standards-bsv/blob/main/.github/CODE_OF_CONDUCT.md
+     - 👷‍♀️ Created small PRs. In most cases this will be possible.
+     - ✅ Provided tests for your changes.
+     - 📝 Used conventional commits.
+     - 📗 Update any related documentation.
+-->
+
+# Pull Request Template
+
+## What type of PR is this?
+
+- [ ] Refactor
+- [ ] Feature
+- [ ] Bug Fix
+- [ ] Security Fix/Improvement
+- [ ] Optimization
+- [ ] Documentation Update
+
+## Description
+
+## Related Tickets & Documents
+
+<!--
+For pull requests that relate or close an issue, please include them
+below.  We like to follow [Github's guidance on linking issues to pull requests](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).
+
+For example having the text: "closes #1234" would connect the current pull
+request to issue 1234.  And when we merge the pull request, Github will
+automatically close the issue.
+-->
+
+- Related Issue #
+- Closes #
```

Comment:
Maybe should be part of the comment above, because we still use JIRA as our first source of tasks, so each time we create a PR, we would need to remove that.
WDYT?